libfreerdp/core/info.c

```diff
@@ -801,6 +797,9 @@ static void rdp_write_info_packet(rdpRdp* rdp, wStream* s)
 		cbPassword = ConvertToUnicode(CP_UTF8, 0, "*", -1, &passwordW, 0) * 2;
 	}
 
+	/* excludes (!) the length of the mandatory null terminator */
+	cbPassword = cbPassword > 0 ? cbPassword - 2 : 0;
```

Comment:
I´d change that to `cbPassword = cbPassword > 2 ? cbPassword - 2 : cbPassword;` to avoid a possible overflow